tests/kafkatest/tests/core/replication_test.py

```diff
@@ -105,7 +105,7 @@ def __init__(self, test_context):
  
     def min_cluster_size(self):
         """Override this since we're adding services outside of the constructor"""
-        return super(ReplicationTest, self).min_cluster_size() + self.num_producers + self.num_consumers
+        return super(ReplicationTest, self).min_cluster_size() + 2
```

Comment:
I'm having trouble understanding this substitution.  How did you decide that 2 was the correct number?  I can't find where `self.num_producers` or `self.num_consumers` should be defined.